web/client/src/App.js

```diff
@@ -0,0 +1,90 @@
+/* eslint "require-jsdoc": ["error", {
+    "require": {
+        "FunctionDeclaration": true,
+        "MethodDefinition": true,
+        "ClassDeclaration": false
+    }
+}]*/
+
+import React, { useEffect } from "react";
+import "./App.css";
+import Credentials from "./components/Credentials";
+import {
+  MDBNavbar,
+  MDBNavbarBrand,
+  MDBContainer,
+  MDBCol,
+  MDBRow,
+} from "mdbreact";
+import { Route, NavLink, HashRouter } from "react-router-dom";
+import Scopes from "./components/Scopes";
+import Button from "./components/Button";
+import { getCacheToken } from "./util/apiWrapper";
+
+/**
+ * @return {HashRouter} returns webapp as a whole
+ */
+function App() {
+  useEffect(() => {
+    const loadSampleResponse = async () => {
+      const sampleBody = JSON.stringify({
+        requesttype: "fetch",
+        args: {
+          "--scope": ["cloud-platform", "userinfo.email"],
+        },
+        needToken: "true",
+        uploadcredentials: {
+          client_id: "random",
+          client_secret: "mock",
+          quota_project_id: "data",
+          refresh_token: "to",
+          type: "use",
+        },
+      });
+
+      const resp = await getCacheToken(sampleBody);
+      console.log(resp); // eslint-disable-line
+    };
+
+    loadSampleResponse();
+  }, []);
+
+  return (
+    <HashRouter>
+      {" "}
+      <div className="App">
+        {" "}
+        <MDBNavbar color="blue">
+          {" "}
+          <MDBNavbarBrand>
+            {" "}
+            <img
+              src={"clogo.png"}
+              width="250"
+              alt="This is a logo for Google Cloud"
+            />{" "}
+          </MDBNavbarBrand>{" "}
+        </MDBNavbar>{" "}
+        <Route exact path="/" component={Credentials} />{" "}
```

Comment:
"Credentials" refers to credentials file input. It is probably better to name the root component something like "Main"